docs/pages/linking/overview.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: Overview of Linking, Deep Links, Android App Links, and iOS Universal Links
+sidebar_title: Overview
+description: An overview of available resources to implement Linking and Deep Links in your Expo apps.
+---
+
+import { AndroidIcon } from '@expo/styleguide-icons/custom/AndroidIcon';
+import { AppleIcon } from '@expo/styleguide-icons/custom/AppleIcon';
+import { BookOpen02Icon } from '@expo/styleguide-icons/outline/BookOpen02Icon';
+
+import { BoxLink } from '~/ui/components/BoxLink';
+import { CODE } from '~/ui/components/Text';
+
+## Linking
+
+Linking allows your app to interact with incoming and outgoing URLs. In this process, the user not only gets directed to open your app, but they are taken to a specific screen (route) within the app.
+
+For example, by clicking an advertisement of your product, your app will open on the user's device and they can view that product's details. This product's link that the user clicked may look like:
+
+```html
```

Comment:
See comment below; this is actually a deep link, I believe.

Reply:
I've moved this example to Deep links section